.github/workflows/update_docs.yml

```diff
@@ -0,0 +1,37 @@
+name: Update Documentation
+
+on:
```

Comment:
Is this missing something? Is this the thing that you didn't get working? If that's the case maybe just move it to another branch?

Reply:
Yeah I took this out cause it doesn't work.
I can move to a different branch.

Reply:
Thanks!